src/NBitcoin/Networks.cs

```diff
@@ -388,11 +389,11 @@ private static Network InitStratisTest()
             messageStart[3] = 0x11;
             var magic = BitConverter.ToUInt32(messageStart, 0); //0x5223570; 
 
-            var genesis = Network.StratisMain.GetGenesis().Clone();
+            var genesis = Network.StratisMain.GetGenesis();
```

Comment:
Why was `.Clone();` removed here? 
And if it was removed here why was it left for `var consensus = Network.StratisMain.Consensus.Clone();` a few lines above?

Reply:
GetGenesis already performs a Clone.

Reply:
Hmm, makes sense, good notice. 

Since GetGenesis() performs a deserialization (cloning basically) I'd suggest renaming that so the name reflects that function will return a copy of object and not the original. Something like 
GetGenesisClone() or GetGenesisDuplicate().

Reply:
That will affect many files. Maybe we can do that in a subsequent PR.

Reply:
Agreed. Better to make a separate PR just with this change.